src/js/backup_restore.js

```diff
@@ -145,7 +145,7 @@ function configuration_backup(callback) {
                 configuration.LED_COLORS = jQuery.extend(true, [], FC.LED_COLORS);
                 configuration.BOARD_ALIGNMENT_CONFIG = jQuery.extend(true, {}, FC.BOARD_ALIGNMENT_CONFIG);
                 configuration.CRAFT_NAME = FC.CONFIG.name;
-                configuration.DISPLAY_NAME = FC.CONFIG.displayName;
+                configuration.PILOT_NAME = FC.CONFIG.pilotName;
```

Comment:
We should not force the new name upon current and older version. There are more places in the PR to retain this principle.

```suggestion
            if (senver,gte(FC.CONFIG.apiVersion, API_VERSION_1.45)) {
                configuration.PILOT_NAME  = FC.CONFIG.pilotName;
            } else {
                configuration.DISPLAY_NAME = FC.CONFIG.displayName;
            }
```